### PR TITLE
added Kirk's edits

### PR DESCRIPTION
**Pull request by ckyriazis**
Adding some edits that @klohmueller sent me.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -273,7 +274,7 @@ \section*{Results}
 models. Figure \ref{fig:cartoon} shows a graphical
 representation of the structure of \stdpopsim. The package centers
 on a catalog of species (Fig.~\ref{fig:cartoon}A), initially consisting of humans,
-\textit{D.~melanogaster}, and \textit{A.~thaliana}.
+\textit{D.~melanogaster}, and \textit{A.~thaliana}. %KEL: we mention 5 species earlier at the end of the Intro. It might be confusing to just highlight 3 here without mentioning the other 2. Or we just mention the main 3 in the Intro and leave teh other 2 as add ons later?
```
Comment (petrelharp) [inline popSimManu.tex]: I think we should update this to include the 5 spp. Why not?

File: popSimManu.tex [hunk 2]
```diff
@@ -293,7 +294,7 @@ \section*{Results}
 the \cite{salome2011recombination} map for \textit{A.~thaliana};
 the \cite{comeron2012many} map for \textit{D.~melanogaster},
 and the \cite{campbell2016pedigree} map for \textit{C.~familiaris}.
-Adding further maps to the library is straightforward.
+Adding further maps to the library is straightforward. %KEL: Should we allude to how to do this?
```
Comment (petrelharp) [inline popSimManu.tex]: It's not documented - right now someone would do it by copying the existing structure for other maps (which should be straightforward) - see the response to reviewers. I think this is fine.

File: popSimManu.tex [hunk 3]
```diff
@@ -374,7 +375,9 @@ \subsection*{The Species Catalog}
 which specifies the out-of-Africa bottleneck as well as the subsequent divergence of
 the European and Asian populations (OutOfAfrica\_3G09), the \cite{tennessen2012evolution} two-population variant of the
 \citeauthor{gutenkunst2009inferring} model which does not include Asian populations, but more explicitly models
-recent rapid human population growth (OutOfAfrica\_2T12), the \cite{browning2018ancestry} admixture model
+recent rapid human population growth in Europe.
+
```
Comment (petrelharp) [inline popSimManu.tex]: whoops on this blank line?

File: popSimManu.tex [hunk 4]
```diff
@@ -475,6 +478,9 @@ \subsection*{Use case: comparing methods of demographic inference}
 Instead, many methods work by matching coalescence time distributions,
 and a single-population model with varying population size can match any coalescence time distribution
 (in which case coalescence rate is the inverse of the effective size).
+%KEL: I think the sentence above is a bit confusing--maybe rephrase as "Instead, many methods work by matching coalescence time distributions,
+%and a single-population model with varying population size can be fit to a coalescence time distribution in data generated from a more complex model including migrattion.
+% In this case, the coalescence rate is the inverse of the effective population size.
```
Comment (petrelharp) [inline popSimManu.tex]: I support rewording the sentence, but this new one doesn't say the same thing: the current wording says something stronger than "can be fit". How about
> Instead, many methods work by matching coalescence time distributions, and a single-population model with varying population size can be fit to a coalescence time distribution in data generated from a more complex model including migration. In this case, the coalescence time distributions will exactly match, and the coalescence rate is the inverse of the effective population size.

File: popSimManu.tex [hunk 5]
```diff
@@ -592,7 +598,7 @@ \subsection*{Use case: comparing methods of demographic inference}
 Africa from which a European population splits off following a bottleneck, with no 
 post-divergence gene flow between the African and European population (Figure \ref{fig:two_popn_fly}A).
 Here again, we find that \dadi and \fastsimcoal infer more consistent histories,
-but ignore the brief population bottleneck in Europe. In addition, \dadi
+but but do not detect the brief bottleneck in Europe, likely due to not including that epoch in the inference. In addition, \dadi
```
Comment (petrelharp) [inline popSimManu.tex]: "not including in the inference" is a big confusing - what do we mean here?

File: popSimManu.tex [hunk 6]
```diff
@@ -627,10 +633,10 @@ \section*{Discussion}
 or in sanity checks, to see if observed data
 (e.g., levels of divergence or the allele frequency spectrum)
 are roughly consistent with the hypothesized scenario.
-Currently, many studies would benefit from simulation-based checks such as these,
-but this presents a major barrier,
+Currently, many studies would benefit from such simulation-based checks.
+However,  there are major barriers to implementation,
 since individual research groups must reimplement complex, previously published demographic models,
-a task made especially daunting by additional layers of realism (e.g., recombination maps).
+ task made especially daunting by additional layers of realism (e.g., recombination maps).
```
Comment (petrelharp) [inline popSimManu.tex]: should still have `a`, I think?

File: popSimManu.tex [hunk 7]
```diff
@@ -920,9 +927,7 @@ \section*{Acknowledgments}
 and the Simons Center for Quantitative Biology at Cold Spring Harbor Laboratory
 for sponsoring the first workshop.
 Early on in the project we were encouraged by many people including Patrick Phillips, Richard Durbin,
-Dmitri Petrov, and Sohini Ramachandran. 
-In addition we would like to thank NESCENT and Matt Hahn, Victoria Sork, and Michael Whitlock for organizing a 2014
-catalysis meeting in which many of the goals of this effort were first laid out. 
+Dmitri Petrov, and Sohini Ramachandran.
```
Comment (petrelharp) [inline popSimManu.tex:951]: looks like we need a rebase

**Comment by petrelharp** [comment]
Thanks! I agree with the `%KEL` comments (but have commented on them); should we use this PR to incorporate those changes? Do you want to do that, @ckyriazis?

**Comment by ckyriazis** [comment]
sounds good! i'll coordinate with kirk to make the additional changes

File: popSimManu.tex (hunk 5 above)
Comment (ckyriazis) [reply popSimManu.tex]: i think what is meant is that there isnt a separate epoch in the dadi or fsc model to account for a bottleneck? how would you suggest better stating that?

File: popSimManu.tex (hunk 6 above)
Comment (ckyriazis) [reply popSimManu.tex]: oops yea

File: popSimManu.tex (hunk 5 above)
Comment (petrelharp) [reply popSimManu.tex]: how about
> infer more consistent histories but do not include Europe's brief bottleneck in their model.

... ?

File: popSimManu.tex (hunk 4 above)
Comment (ckyriazis) [reply popSimManu.tex]: is "in data" right here? sounds maybe a bit strange to me. perhaps "using data"?

File: popSimManu.tex (hunk 5 above)
Comment (ckyriazis) [reply popSimManu.tex]: hmm this doesnt seem to quite convey the point that the inference ignores it because the model assumed a priori doesnt really allow for the model to separately infer a bottleneck pop size parameter. how about: 

> but do not detect the brief bottleneck in Europe, likely due to the constraints of the inference model.

**Comment by ckyriazis** [comment]
just pushed another commit trying to address the issues above and deal with merge conflicts. let me know if there are any remaining issues.

File: popSimManu.tex (hunk 4 above)
Comment (petrelharp) [reply popSimManu.tex]: Good point, that's a bad sentence. How about:

> a single-population model with varying population size can be fit to data generated from a more complex model including migration.

File: popSimManu.tex (hunk 5 above)
Comment (petrelharp) [reply popSimManu.tex]: Hm, ok. Can this be more definitive? If the models just plain don't allow a bottleneck, then it's not "likely" due to the constraints, it's definately because of the constraints; and we should say *what* constraints (ie, the models just don't have a bottleneck)?

**Comment by petrelharp** [comment]
Thanks! Two more comments...

**Comment by ckyriazis** [comment]
one more commit...hoping that takes care of things so we can merge before others start working on the manuscript

**Comment by petrelharp** [comment]
Great, thanks!!
